rotkehlchen/db/settings.py

```diff
@@ -306,3 +317,96 @@ def serialize_db_setting(
         else:
             value = [x.serialize() for x in value]
     return value
+
+
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class DBSettingsUnfrozen:
```

Comment:
No no no. Why add this? Just use one class and have it unfrozen. This is silly duplication.

Reply:
Yes, it's very bad. Since I'm not familiar with the entire codebase i don't know if the DBSettings dataclass has a reason to be frozen (as it is now). If not, then I can just mark it as frozen=False and use that. I tried to avoid the duplication with inheritance but in datclasses you can't have unforzen child and frozen parent class. I tried a different approach too but didn't work as expected. So  Ileft this duplicaiton and see how to handle it better after dicsussion. The best way is to make teh DBSettings frozen=False.

Reply:
please be asking question instead of making assumptions. Thigns will go much faster.

Reply:
So, should I make the DBSettings unfrozen?